coordinator/src/test/scala/filodb.coordinator/NodeCoordinatorActorSpec.scala

```diff
@@ -141,7 +141,8 @@ class NodeCoordinatorActorSpec extends ActorTest(NodeCoordinatorActorSpec.getNew
       dataset1.ref
     }
 
-    it("should return UnknownDataset if attempting to query before ingestion set up") {
+    // Disabling this test as querying for unknown dataset will keep bouncing in the current setup.
+    ignore("should return UnknownDataset if attempting to query before ingestion set up") {
```

Comment:
Can we leave a comment in TempQueryRouter to enable this test back again when workaround is removed?

Reply:
Done in 7eec2ee